subxt/tests/integration/storage.rs

```diff
@@ -79,7 +79,7 @@ async fn storage_n_mapish_key_is_properly_created(
     };
 
     // This is what the generated code hashes a `session().key_owner(..)` key into:
-    let actual_key_bytes = KeyOwner(KeyTypeId([1, 2, 3, 4]), vec![5u8, 6, 7, 8])
+    let actual_key_bytes = KeyOwner(&KeyTypeId([1, 2, 3, 4]), &[5u8, 6, 7, 8])
```

Comment:
Nice!

Reply:
Thanks! Having the API as reference does improve the ergonomics IMO :D